koku/api/deprecated_settings/view.py

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
Where did this file come from?

Reply:
Probably a bad merge. It's been a bit of an ordeal keeping this branch up to date.

Reply:
So are we removing this file? or...

Reply:
Yes.